### ISSUE TITLE
helpers/memenv/memenv_test.cc: Use correct datatype

### ISSUE THREAD

**Issue by GoogleCodeExporter** [chrsmithdemos/leveldb#41]
```
While running `make check` you'll most likely run into a build error since 
file_size expects uint64_t but seems to be size_t. Here's the diff to make it 
build again:

diff --git a/helpers/memenv/memenv_test.cc b/helpers/memenv/memenv_test.cc      

index 30b0bb0..3791dc3 100644                                                   

--- a/helpers/memenv/memenv_test.cc                                             

+++ b/helpers/memenv/memenv_test.cc                                             

@@ -26,7 +26,7 @@ class MemEnvTest {                                            

 };                                                                                                                                                       

 TEST(MemEnvTest, Basics) {                                                                                                                               
-  size_t file_size;                                                            

+  uint64_t file_size;                                                          

   WritableFile* writable_file;                                                                                                                           
   std::vector<std::string> children; 
```

---

Original issue reported on code.google.com by `jo...@bergstroem.nu` on 26 Sep 2011 at 9:08

**Comment by GoogleCodeExporter** [comment]
```
Whoops. Will fix.
```

---

Original comment by `h...@chromium.org` on 26 Sep 2011 at 4:15
- Changed state: **Started**

**Comment by GoogleCodeExporter** [comment]
```
Fixed in Git 26db4d971a15 / SVN r52.
```

---

Original comment by `h...@chromium.org` on 26 Sep 2011 at 4:49
- Changed state: **Fixed**
